src/compiler/checker.ts

```diff
@@ -11193,7 +11193,10 @@ namespace ts {
             let hasComputedStringProperty = false;
             let hasComputedNumberProperty = false;
 
-            for (const memberDecl of node.properties) {
+            let i = 0;

```

Comment:
This looks like it can probably be part of the following `for` declaration rather than on its own.